content/api/entity-reference/gift/index.html

```diff
@@ -123,12 +123,12 @@
             <tr>
                 <td>gift_aid_amount</td>
                 <td><a href="{{ stache.config.constituent_entity_reference }}#Currency">currency</a></td>
-                <td>The amount of tax reclaimed from Gift Aid for this gift. For UK only.</td>
+                <td>The amount of tax reclaimed from Gift Aid for this gift.  For UK formatted databases only. This is a computed field based on total qualified gift aid amount across all <code>gift_splits</code>.</td>
             </tr>
             <tr>
                 <td>gift_aid_qualification_status</td>
                 <td>string</td>
-                <td>The Gift Aid qualification status of the gift. Available values are: qualified, notqualified, and partlyqualified. For UK only.</td>
+                <td>The Gift Aid qualification status of the gift. Available values are: qualified, notqualified, and partlyqualified. For UK formatted databases only. This is a computed field based on tax declaration information and database format.</td>
```

Comment:
The reasoning for the "for UK formatted databases only" is that you can have a database in Russia and have it "format as UK" and then this field will apply to that database. So for UK only isn't entirely correct. I'm more than happy to make the change across all entity references if you're ok with the wording. "For UK-formatted databases only"